tools/clang/test/CodeGenHLSL/quick-test/regression_github_1724.hlsl

```diff
@@ -0,0 +1,17 @@
+// RUN: %dxc /T vs_6_0 /E main %s
```

Comment:
I could name this file differently but for compiler crashes, I think it'd make sense to just have regression tests with the bug ID since the point is not really to test any functionality. @hekota Thoughts?

Reply:
The file name is fine, but I would add a simple `CHECK: ret void` or something similar to make sure the compiler succeeded and generated some code. The reason for this is that some of the crashes are 'silent' and do not report an error.

Reply:
I added a CHECK and renamed the file.